script/script.js

```diff
@@ -14,6 +14,34 @@ function clearTimeOuts()
     clearTimeout(timeOut2);
 }
 
+
+//Push names in the cards
+function pushPlayer1()
+{
+    //Get name from input field
+    let player1Name =  $("#player1Input").val();
+
+    //Validate and push names
+    if(player1Name != "")
+    {
+        $("#player1Name").text(player1Name);
+        $("#player1Input").val("");
+    }
+}
+function pushPlayer2()
+{
+    //Get name from input field
+    let player2Name = $("#player2Input").val();
+
+    //Validate and push names
+    if(player2Name != "")
+    {
+        $("#player2Name").text(player2Name);
+        $("#player2Input").val("");
+    }
+}

```

Comment:
We can shorten these two methods into a single method

Reply:
Alright I will work on this issue and create another pull request